apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -118,14 +118,36 @@
     Boolean isEmailUsernameEnabled = false;
     String usernameLabel = "username";
 
+    Boolean isSelfSignUpEnabledInTenant = false;
+    Boolean isUsernameRecoveryEnabledInTenant = false;
+    Boolean isPasswordRecoveryEnabledInTenant = false;
+    Boolean isMultiAttributeLoginEnabledInTenant = false;
```

Comment:
false is redundant

Reply:
Fixed in 1591ee73e98c6530a217973e9acc72fb277be5af